drafts/str4d-overwinter-activation/draft1.rst

```diff
@@ -0,0 +1,229 @@
+::
+
+  ZIP: 200
+  Title: Network Upgrade Mechanism
+  Author: Jack Grigg <jack@z.cash>
+  Category: Process
+  Created: 2018-01-08
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST", "SHOULD", "SHOULD NOT", and "MAY" in this document are to be interpreted as
+described in RFC 2119. [#RFC2119]_
+
+The terms below are to be interpreted as follows:
+
+Branch
```

Comment:
see my comment here https://github.com/bitcartel/zips/commit/c8405e781a1f8305188a48b30fb5b2d24dce7efb#r27449112
about the definition of branch.
Does soft forking make it a different branch?
This should be clarified.

Reply:
Soft forks do not change the consensus rules and are not bilateral (old nodes will reorg to the soft-forked chain), so no, a new branch is not created.